### DISCUSSION TITLE
Logo margin and scrolling

### DISCUSSION THREAD

**Discussion by treeandcoffee** [GetPublii/Publii#1883] [Help]
Hi
I use the premium theme lignes. My rectangular logo is displayed in the navbar and I think it is in the top__inner container. The logo size is good. But the distance/margin from the top edge to the logo and from the logo to the spacing line at the bottom is too small. If this could be corrected with
`.logo>img {
    margin: 10% auto;
}`
then margin is good, but when I scroll down, the nav bar does not disappear completely like it should, but it remains around the specified margin. This means that the nav bar still appears. 

Do you have any idea how I can solve this?

**Comment by candidexmedia** [comment]
Have you solved your issue? Do you have a live site that we can look at?

**Comment by treeandcoffee** [comment]
Many thanks @candidexmedia, yes it is solved by changing .navbar .navbar__menu variables in custom CSS, thanks to @bobmitro.
